sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql

```diff
@@ -45,286 +45,238 @@ desktop_data_bing AS (
     `mozdata.search.search_clients_engines_sources_daily`
   WHERE
     submission_date = @submission_date
-    AND distribution_id NOT LIKE '%acer%'
+    AND client_id NOT IN (SELECT client_id FROM `mozdata.analysis.acer_cohort`)
```

Comment:
Check against the cohort is here -- I'm assuming we no longer also need to remove distribution_id LIKE '%acer%' because those people would all be in the cohort?

Reply:
The client list makes sure us from including the previous firefox users on Acer distrubtion, but not prevent us from the future ones (in case there is any), so let's keep it in there.